extensions/display-modes/src/utils.ts

```diff
@@ -0,0 +1,76 @@
+import { environment, getPreferenceValues, Icon, Image, Keyboard, List } from "@raycast/api";
+import { execa, ExecaError } from "execa";
+import { chmod } from "fs/promises";
+import { join } from "path";
+import { DisplayInfo, Mode } from "./types";
+
+const preferences: Preferences = getPreferenceValues();
+
```

Comment:
```suggestion
import { environment } from "@raycast/api";
import { execa } from "execa";
import { chmod } from "fs/promises";
import { join } from "path";
import { DisplayInfo, Mode } from "./types";

```
Just removed unused imports

Reply:
Done.